internal/retry.go

```diff
@@ -17,6 +17,7 @@ package internal
 import (
 	"context"
 	"fmt"
+	"google.golang.org/grpc/codes"
```

Comment:
This should be down with the other imports. Please run `gofmt -w internal/retry.go` that should fix it.

Reply:
Fixed it myself, needed to use `goimports` instead, my bad.